powerhome/playbook
# [CHMN-77] Add close on click "outside" prop to passphrase popover

#### Screens

No new visuals.

#### Breaking Changes

No, additive changes to the passphrase kit. Other kits not affected. Passphrase kit not used yet (but will be soon)

#### Runway Ticket URL

[Link to Runway Story](https://nitro.powerhrg.com/runway/backlog_items/CHMN-77)

#### How to test this

/kits/passphrase/react
/kits/passphrase/

See that the tips popover closes on click outside of it

#### Checklist:


*The normal release cut off deadline is 3p EDT each week. Please reach out to the release team if you have an urgent request or need a release off cycle.*
